src/mapgen/mg_ore.cpp

```diff
@@ -498,7 +498,7 @@ void OreVein::generate(MMVManip *vm, int mapseed, u32 blockseed,
 		}
 
 		// randval ranges from -1..1
-		float randval   = (float)pr.next() / (pr.RANDOM_RANGE / 2) - 1.f;
+		float randval   = (float)pr.next() / ((float)pr.RANDOM_RANGE / 2.f) - 1.f;
```

Comment:
shouldn't the cast happen after the calculation
otherwise the result is not guaranteed to be the same (no idea if that edge case is relevant here)

Reply:
well I don't think it matters, this is just ore generation, decided to do it this way so the comment above is actually accurate (before the range was ~[-1, 1.00006]).

Reply:
It matters because the mapgen is supposed to be deterministic, even across versions.

Reply:
ok did that and added a comment explaining.